CMakeLists.txt

```diff
@@ -385,6 +385,14 @@ else()
   add_definitions( -DBOOST_UBLAS_MOVE_SEMANTICS )
 endif ()
 
+# std::atomic_ref is part of C++20, hence using boost::atomic_ref
+# boost::atomic_ref has full support from version 1.74
```

Comment:
overall ok to me...but i am wondering if we should not make it forward compatible to
c++20, by defining a kratos atomic_ref.

what do you think?

Reply:
Look at the code, this is basically what I did :)

Reply:
well, i may misunderstand the code, but i  would say that you always require boost 1.74, even if c++20 is being used...

Reply:
ah got it, good catch